parsers/ENTSOE.py

```diff
@@ -148,8 +190,6 @@ def query_ENTSOE(session, params, target_datetime=None, span=(-48, 24)):
     if target_datetime is None :
         target_datetime = arrow.utcnow()
     else:
-        # for a specific datetime, fetch 24h values starting from target datetime
-        span = (-1, 48)
```

Comment:
is there a reason to remove this? Changing it in to span = (-48, 1) in a PR (soon)

Reply:
Alright, it's because of `query_production_per_unit`.
Then I suggest changing to (-24, 0) whenever we call with `target_datetime` (and changing values in `feeder.lib.fetch_data.REFETCH_SUPPORT` to `1D` instead of `2D` for ENTSO).

Reply:
Why force the span inside the code? I find that counter-intuitive given that it's also given as a function argument (so it shouldn't be silently overridden).
Any reason to have it override?

Reply:
the reason was that `query_entsoe` is called in many places. So that when fetching for a `target_datetime`, we didn't have to specify the span since it's always the same. But I agree that it's not explicit, and Guido says `explicit is better than implicit`.